pkg/plugin/testhelper/pem.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package testhelper provides commonly used test functions in different packages.
+package testhelper
```

Comment:
nit: testutil? I have only see testutil in our repos, maybe keep the convention?

Reply:
I would try and follow the stdlib here and name this package `keyutil` or something.

Reply:
I was naming this after pmap: https://github.com/abcxyz/pmap/tree/main/internal/testhelper

Reply:
Renamed to `keyutil`, and also moved `ReadRSAPrivateKey` under `keyutil`.